.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,11 @@
+## Changelog
+
+- add some bullet points
+
+## Code Quality Assurance
+
+- [ ] All tests pass locally.
+- [ ] All tests pass in CI.
+- [ ] Coverage is above 90%.
+- [ ] All exposed functions, interfaces, and types have documentation.
+- [ ] Run benchmarks again if there is functionality change.
```

Comment:
Add complexity checks too. ([see](https://hackernoon.com/golang-linters-how-they-analyze-code-formatting-complexity-and-comparison))